components/gitpod-db/src/typeorm/migration/1612448906950-ExampleRepositories.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+import {MigrationInterface, QueryRunner} from "typeorm";
+
+export class ExampleRepositories1612448906950 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        const oldEntries = [
+            { url: 'https://github.com/gitpod-io/django-locallibrary-tutorial', description: '**Python** - Tutorial "Local Library" website written in Django', priority: 10 },
+            { url: 'https://github.com/gitpod-io/gs-spring-boot.git', description: '**Java** - Building an Application with Spring Boot', priority: 9 },
+            { url: 'https://github.com/gitpod-io/NextSimpleStarter.git', description: '**JavaScript** - Simple PWA boilerplate with Next.js and Redux', priority: 8 },
```

Comment:
**thought**: I'm going to miss this one. This was a good, simple and fast enough to start.

Reply:
@gtsiolis Excellent point, thanks! The current batch of examples are an attempt to use more "real-world" / "complex" projects to really demonstrate what Gitpod can do. So some of our examples are being switched to this type of project featuring a "real-world" dev setup.

However, I know @svenefftinge has been advocating for more "well-known language starter projects", like NextSimpleStarter and go-gin-app. My thought was that these projects are too simple (and don't benefit at all from prebuilds), presenting Gitpod as "just another online coding sandbox". But I'm starting to hesitate a bit.

My hope is that, with this experiment, we'll learn more about how users discover & learn to use Gitpod, which will inform another iteration of even better examples in not too long.